codalab/lib/upload_manager.py

```diff
@@ -148,7 +171,14 @@ def write_git_repo(self, source: str, bundle_path: str):
         file_util.git_clone(source, bundle_path)
 
     def write_fileobj(
-        self, source_ext: str, source_fileobj: IO[bytes], bundle_path: str, unpack_archive: bool
+        self,
+        source_ext: str,
+        source_fileobj: IO[bytes],
+        bundle_path: str,
+        unpack_archive: bool,
+        bundle_conn_str=None,
+        index_conn_str=None,
+        progress_callback=None,
```

Comment:
Is it OK that we're not using progress_callback in this method?

Reply:
Yes, the default progress_callback function is set to None. But we need `progress_callback` in the function signature, since the ClientUploadManager need to know the upload progress by `_write_fileobj`